  INFO root [2019-03-02 09:58:07,777] - 
  INFO root [2019-03-02 09:58:07,778] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 09:58:07,779] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 09:58:07,779] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 09:58:07,779] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 09:58:07,784] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:13416 )
  INFO root [2019-03-02 09:58:07,784] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:13416, SOMAXCONN=5.
  INFO root [2019-03-02 09:58:07,784] - server-configs:
  INFO root [2019-03-02 09:58:07,784] - 	internalTcpAddr : 192.168.0.148:13416
  INFO root [2019-03-02 09:58:07,784] - 	componentID : 5000
  WARN root [2019-03-02 09:58:07,787] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 09:58:07,851] - Components::findLogger: found logger, addr:192.168.0.148:13264
  INFO root [2019-03-02 09:58:07,868] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 09:58:07,868] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 09:58:07,875] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 14204 ----
  INFO root [2019-03-02 09:58:07,875] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 09:58:07,875] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 09:58:07,876] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 09:58:08,204] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 09:58:08,204] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 09:58:14,355] - Forwarding logs to logger[192.168.0.148:13264/0/0/0]...
  INFO root [2019-03-02 10:02:13,260] - 
  INFO root [2019-03-02 10:02:13,276] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:02:13,276] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:02:13,276] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:02:13,276] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:02:13,276] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:1720 )
  INFO root [2019-03-02 10:02:13,276] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:1720, SOMAXCONN=5.
  INFO root [2019-03-02 10:02:13,276] - server-configs:
  INFO root [2019-03-02 10:02:13,276] - 	internalTcpAddr : 192.168.0.148:1720
  INFO root [2019-03-02 10:02:13,276] - 	componentID : 5000
  WARN root [2019-03-02 10:02:13,276] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:02:14,792] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 10236 ----
  INFO root [2019-03-02 10:02:14,792] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:02:14,792] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:02:14,792] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:02:15,135] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:02:15,135] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:02:22,464] - Components::findComponents: find logger(1)...
  WARN root [2019-03-02 10:02:23,652] - Components::findComponents: not found logger!
  INFO root [2019-03-02 10:02:24,667] - Components::findComponents: find dbmgr(1)...
  INFO root [2019-03-02 10:02:26,792] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:02:28,870] - Components::findComponents: find dbmgr(3)...
  INFO root [2019-03-02 10:02:30,932] - Components::findComponents: find dbmgr(4)...
  INFO root [2019-03-02 10:02:32,995] - Components::findComponents: find dbmgr(5)...
  INFO root [2019-03-02 10:02:35,073] - Components::findComponents: find dbmgr(6)...
  INFO root [2019-03-02 10:02:37,151] - Components::findComponents: find dbmgr(7)...
  INFO root [2019-03-02 10:02:39,197] - Components::findComponents: find dbmgr(8)...
  INFO root [2019-03-02 10:02:41,275] - Components::findComponents: find dbmgr(9)...
  INFO root [2019-03-02 10:02:43,353] - Components::findComponents: find dbmgr(10)...
  INFO root [2019-03-02 10:02:45,447] - Components::findComponents: find dbmgr(11)...
  INFO root [2019-03-02 10:02:47,574] - Components::findComponents: find dbmgr(12)...
  INFO root [2019-03-02 10:02:49,668] - Components::findComponents: find dbmgr(13)...
  INFO root [2019-03-02 10:02:51,746] - Components::findComponents: find dbmgr(14)...
  INFO root [2019-03-02 10:02:53,824] - Components::findComponents: find dbmgr(15)...
  INFO root [2019-03-02 10:02:55,901] - Components::findComponents: find dbmgr(16)...
  WARN root [2019-03-02 10:02:57,995] - Components::findComponents: find dbmgr(17)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:00,073] - Components::findComponents: find dbmgr(18)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:02,135] - Components::findComponents: find dbmgr(19)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:04,213] - Components::findComponents: find dbmgr(20)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:06,291] - Components::findComponents: find dbmgr(21)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:08,372] - Components::findComponents: find dbmgr(22)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:10,450] - Components::findComponents: find dbmgr(23)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:12,528] - Components::findComponents: find dbmgr(24)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:14,606] - Components::findComponents: find dbmgr(25)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:16,684] - Components::findComponents: find dbmgr(26)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:18,762] - Components::findComponents: find dbmgr(27)...
delay time is too long, please check the dbmgr logs!
  INFO root [2019-03-02 10:03:39,058] - 
  INFO root [2019-03-02 10:03:39,058] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:03:39,073] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:03:39,073] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:03:39,073] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:03:39,073] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:1766 )
  INFO root [2019-03-02 10:03:39,073] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:1766, SOMAXCONN=5.
  INFO root [2019-03-02 10:03:39,073] - server-configs:
  INFO root [2019-03-02 10:03:39,073] - 	internalTcpAddr : 192.168.0.148:1766
  INFO root [2019-03-02 10:03:39,073] - 	componentID : 5000
  WARN root [2019-03-02 10:03:39,073] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:03:40,604] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 12088 ----
  INFO root [2019-03-02 10:03:40,604] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:03:40,604] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:03:40,604] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:03:40,995] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:03:40,995] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:03:48,341] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:03:48,388] - Components::findComponents: found logger, addr:192.168.0.148:1763
  INFO root [2019-03-02 10:03:48,388] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:03:48,388] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:03:48,434] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:03:48,497] - Components::findComponents: found dbmgr, addr:192.168.0.148:1768
  INFO root [2019-03-02 10:03:48,497] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:03:48,497] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:03:48,497] - Components::findComponents: found dbmgr, addr:192.168.0.148:1768
  WARN root [2019-03-02 10:03:49,512] - Forwarding logs to logger[192.168.0.148:1763/0/0/0]...
 ERROR root [2019-03-02 10:09:01,266] - Components::removeComponentByChannel: cellapp : 8001, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

 ERROR root [2019-03-02 10:09:01,268] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:09:01,269] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:09:01,270] - ==> [2019-03-02 10:09:01 265] TCPPacketReceiver::processPendingEvents(192.168.0.148:1795): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:09:01,270] - ==> [2019-03-02 10:09:01 266] Components::removeComponentByChannel: cellapp : 8001, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:09:01,271] - ==> [2019-03-02 10:09:01 268] TCPPacketReceiver::processPendingEvents(192.168.0.148:1763): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:09:01,271] - ==> [2019-03-02 10:09:01 268] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:09:01,271] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1784): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:09:01,272] - Components::removeComponentByChannel: loginapp : 9000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:09:01,273] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1768): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:09:01,273] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:09:01,292] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:09:01,293] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:09:01,293] - Shutdowner::onShutdownEnd: shutting down
  WARN root [2019-03-02 10:09:01,294] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1788): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-03-02 10:09:01,294] - Components::removeComponentByChannel: cellappmgr : 6000, Normal exit!
  WARN root [2019-03-02 10:09:01,295] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1790): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  WARN root [2019-03-02 10:09:01,295] - Baseappmgr::onChannelDeregister: erase baseapp[7001], currsize=0
  INFO root [2019-03-02 10:09:01,296] - Components::removeComponentByChannel: baseapp : 7001, Normal exit!
 DEBUG root [2019-03-02 10:09:01,297] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-03-02 10:09:01,601] - ThreadPool::destroy(): waiting for thread(2)[0x1f9b18,0x1f9228,], try=1
 DEBUG root [2019-03-02 10:09:02,003] - TPThread::~TPThread(): 0x1f9b18
 DEBUG root [2019-03-02 10:09:02,003] - TPThread::~TPThread(): 0x1f9228
 DEBUG root [2019-03-02 10:09:02,003] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-03-02 10:09:02,031] - Bundle::destroyObjPool(): size 2.
 DEBUG root [2019-03-02 10:09:02,031] - Channel::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 10:09:02,031] - TCPPacket::destroyObjPool(): size 14.
 DEBUG root [2019-03-02 10:09:02,031] - UDPPacket::destroyObjPool(): size 4.
 DEBUG root [2019-03-02 10:09:02,032] - EndPoint::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 10:09:02,032] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 10:09:02,032] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 10:09:02,032] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-03-02 10:09:02,032] - baseappmgr(5000) has shut down.
  INFO root [2019-03-02 10:09:03,035] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-03-02 10:10:26,942] - 
  INFO root [2019-03-02 10:10:26,942] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:10:26,944] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:10:26,945] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:10:26,945] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:10:26,949] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2699 )
  INFO root [2019-03-02 10:10:26,949] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2699, SOMAXCONN=5.
  INFO root [2019-03-02 10:10:26,949] - server-configs:
  INFO root [2019-03-02 10:10:26,950] - 	internalTcpAddr : 192.168.0.123:2699
  INFO root [2019-03-02 10:10:26,950] - 	componentID : 5000
  WARN root [2019-03-02 10:10:26,953] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:10:27,008] - Components::findLogger: found logger, addr:192.168.0.123:2683
  INFO root [2019-03-02 10:10:27,009] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:10:27,009] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:10:27,013] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 2836 ----
  INFO root [2019-03-02 10:10:27,013] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:10:27,013] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:10:27,013] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:10:27,329] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:10:27,329] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 10:10:33,460] - Forwarding logs to logger[192.168.0.123:2683/0/0/0]...
 ERROR root [2019-03-02 10:13:34,109] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:13:34,109] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:13:34,109] - ==> [2019-03-02 10:13:34 112] TCPPacketReceiver::processPendingEvents(192.168.0.123:2683): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:13:34,109] - ==> [2019-03-02 10:13:34 113] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:13:34,139] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2685): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:13:34,139] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:13:34,149] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:13:34,149] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:13:34,149] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:13:34,149] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:16:41,223] - 
  INFO root [2019-03-02 10:16:41,223] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:16:41,223] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:16:41,229] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:16:41,229] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:16:41,232] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2846 )
  INFO root [2019-03-02 10:16:41,232] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2846, SOMAXCONN=5.
  INFO root [2019-03-02 10:16:41,232] - server-configs:
  INFO root [2019-03-02 10:16:41,232] - 	internalTcpAddr : 192.168.0.123:2846
  INFO root [2019-03-02 10:16:41,232] - 	componentID : 5000
  WARN root [2019-03-02 10:16:41,242] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:16:42,753] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 4316 ----
  INFO root [2019-03-02 10:16:42,753] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:16:42,753] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:16:42,753] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:16:43,122] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:16:43,122] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:16:50,310] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:16:50,327] - Components::findComponents: found logger, addr:192.168.0.123:2851
  INFO root [2019-03-02 10:16:50,327] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:16:50,327] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:16:50,328] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:16:50,429] - Components::findComponents: found dbmgr, addr:192.168.0.123:2850
  INFO root [2019-03-02 10:16:50,429] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:16:50,429] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:16:50,430] - Components::findComponents: found dbmgr, addr:192.168.0.123:2850
  WARN root [2019-03-02 10:16:51,431] - Forwarding logs to logger[192.168.0.123:2851/0/0/0]...
 ERROR root [2019-03-02 10:17:07,851] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:17:07,851] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:17:07,851] - ==> [2019-03-02 10:17:07 854] TCPPacketReceiver::processPendingEvents(192.168.0.123:2851): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:17:07,858] - ==> [2019-03-02 10:17:07 856] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:17:07,878] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2850): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:17:07,878] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:17:07,888] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:17:07,888] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:17:07,888] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:17:07,888] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:17:15,719] - 
  INFO root [2019-03-02 10:17:15,728] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:17:15,728] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:17:15,728] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:17:15,728] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:17:15,732] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2901 )
  INFO root [2019-03-02 10:17:15,732] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2901, SOMAXCONN=5.
  INFO root [2019-03-02 10:17:15,732] - server-configs:
  INFO root [2019-03-02 10:17:15,732] - 	internalTcpAddr : 192.168.0.123:2901
  INFO root [2019-03-02 10:17:15,732] - 	componentID : 5000
  WARN root [2019-03-02 10:17:15,739] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:17:17,259] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 11524 ----
  INFO root [2019-03-02 10:17:17,259] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:17:17,259] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:17:17,259] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:17:17,688] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:17:17,688] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:17:24,819] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:17:24,844] - Components::findComponents: found logger, addr:192.168.0.123:2899
  INFO root [2019-03-02 10:17:24,844] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:17:24,844] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:17:24,845] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:17:24,949] - Components::findComponents: found dbmgr, addr:192.168.0.123:2902
  INFO root [2019-03-02 10:17:24,949] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:17:24,950] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:17:24,950] - Components::findComponents: found dbmgr, addr:192.168.0.123:2902
  WARN root [2019-03-02 10:17:25,950] - Forwarding logs to logger[192.168.0.123:2899/0/0/0]...
 ERROR root [2019-03-02 10:18:33,694] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:18:33,695] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:18:33,696] - ==> [2019-03-02 10:18:33 694] TCPPacketReceiver::processPendingEvents(192.168.0.123:2899): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:18:33,696] - ==> [2019-03-02 10:18:33 695] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:18:33,731] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2902): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:18:33,733] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:18:33,739] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:18:33,743] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:18:33,743] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:18:33,743] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:20:36,608] - 
  INFO root [2019-03-02 10:20:36,612] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:20:36,612] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:20:36,612] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:20:36,612] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:20:36,628] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2989 )
  INFO root [2019-03-02 10:20:36,628] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2989, SOMAXCONN=5.
  INFO root [2019-03-02 10:20:36,628] - server-configs:
  INFO root [2019-03-02 10:20:36,628] - 	internalTcpAddr : 192.168.0.123:2989
  INFO root [2019-03-02 10:20:36,628] - 	componentID : 5000
  WARN root [2019-03-02 10:20:36,632] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  WARN root [2019-03-02 10:20:37,248] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20635, errorno=10048
  INFO root [2019-03-02 10:20:37,248] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 10956 ----
  INFO root [2019-03-02 10:20:37,248] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:20:37,252] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:20:37,252] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:20:37,700] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:20:37,707] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 10:20:41,468] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20412, errorno=10048
  INFO root [2019-03-02 10:20:45,628] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:20:45,692] - Components::findComponents: found logger, addr:192.168.0.123:2993
  INFO root [2019-03-02 10:20:45,692] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:20:45,692] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:20:45,702] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:20:45,811] - Components::findComponents: found dbmgr, addr:192.168.0.123:2988
  INFO root [2019-03-02 10:20:45,811] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:20:45,811] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:20:45,818] - Components::findComponents: found dbmgr, addr:192.168.0.123:2988
  WARN root [2019-03-02 10:20:46,828] - Forwarding logs to logger[192.168.0.123:2993/0/0/0]...
 ERROR root [2019-03-02 10:27:41,767] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:27:41,767] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:27:41,767] - ==> [2019-03-02 10:27:41 769] TCPPacketReceiver::processPendingEvents(192.168.0.123:2993): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:27:41,767] - ==> [2019-03-02 10:27:41 769] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:27:41,817] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2988): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:27:41,817] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:27:41,827] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:27:41,827] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:27:41,827] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:27:41,827] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:35:36,386] - 
  INFO root [2019-03-02 10:35:36,386] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:35:36,386] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:35:36,386] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:35:36,390] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:35:36,396] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:3247 )
  INFO root [2019-03-02 10:35:36,396] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:3247, SOMAXCONN=5.
  INFO root [2019-03-02 10:35:36,396] - server-configs:
  INFO root [2019-03-02 10:35:36,396] - 	internalTcpAddr : 192.168.0.123:3247
  INFO root [2019-03-02 10:35:36,396] - 	componentID : 5000
  WARN root [2019-03-02 10:35:36,399] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:35:37,908] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 2824 ----
  INFO root [2019-03-02 10:35:37,908] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:35:37,908] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:35:37,908] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:35:38,250] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:35:38,250] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 10:35:42,023] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20149, errorno=10048
  INFO root [2019-03-02 10:35:46,166] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:35:46,267] - Components::findComponents: found logger, addr:192.168.0.123:3254
  INFO root [2019-03-02 10:35:46,268] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:35:46,268] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:35:46,269] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:35:46,372] - Components::findComponents: found dbmgr, addr:192.168.0.123:3251
  INFO root [2019-03-02 10:35:46,372] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:35:46,373] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:35:46,373] - Components::findComponents: found dbmgr, addr:192.168.0.123:3251
  WARN root [2019-03-02 10:35:47,374] - Forwarding logs to logger[192.168.0.123:3254/0/0/0]...
 ERROR root [2019-03-02 10:46:59,094] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:46:59,094] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:46:59,094] - ==> [2019-03-02 10:46:59 099] TCPPacketReceiver::processPendingEvents(192.168.0.123:3254): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:46:59,094] - ==> [2019-03-02 10:46:59 101] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:46:59,135] - TCPPacketReceiver::processPendingEvents(192.168.0.123:3251): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:46:59,144] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:46:59,144] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:46:59,144] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:46:59,144] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:46:59,144] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:48:41,754] - 
  INFO root [2019-03-02 10:48:41,754] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:48:41,754] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:48:41,754] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:48:41,754] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:48:41,754] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:3601 )
  INFO root [2019-03-02 10:48:41,754] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:3601, SOMAXCONN=5.
  INFO root [2019-03-02 10:48:41,754] - server-configs:
  INFO root [2019-03-02 10:48:41,754] - 	internalTcpAddr : 192.168.0.123:3601
  INFO root [2019-03-02 10:48:41,754] - 	componentID : 5000
  WARN root [2019-03-02 10:48:41,764] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:48:43,284] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 11596 ----
  INFO root [2019-03-02 10:48:43,284] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:48:43,284] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:48:43,284] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:48:43,698] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 10:48:43,698] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:48:50,825] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:48:50,847] - Components::findComponents: found logger, addr:192.168.0.123:3603
  INFO root [2019-03-02 10:48:50,847] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:48:50,848] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:48:50,849] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:48:50,951] - Components::findComponents: found dbmgr, addr:192.168.0.123:3606
  INFO root [2019-03-02 10:48:50,951] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:48:50,951] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:48:50,958] - Components::findComponents: found dbmgr, addr:192.168.0.123:3606
  WARN root [2019-03-02 10:48:51,958] - Forwarding logs to logger[192.168.0.123:3603/0/0/0]...
 ERROR root [2019-03-02 11:24:21,573] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 11:24:21,575] - The following logs sent to logger failed:
  WARN root [2019-03-02 11:24:21,575] - ==> [2019-03-02 11:24:21 570] TCPPacketReceiver::processPendingEvents(192.168.0.123:3603): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 11:24:21,576] - ==> [2019-03-02 11:24:21 574] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 11:24:21,648] - TCPPacketReceiver::processPendingEvents(192.168.0.123:3606): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 11:24:21,649] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 11:24:21,651] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 11:24:21,651] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 11:24:21,651] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 11:24:21,652] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 14:44:26,549] - 
  INFO root [2019-03-02 14:44:26,559] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 14:44:26,582] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 14:44:26,582] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 14:44:26,582] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 14:44:26,626] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:13606 )
  INFO root [2019-03-02 14:44:26,627] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:13606, SOMAXCONN=5.
  INFO root [2019-03-02 14:44:26,627] - server-configs:
  INFO root [2019-03-02 14:44:26,627] - 	internalTcpAddr : 192.168.0.148:13606
  INFO root [2019-03-02 14:44:26,627] - 	componentID : 5000
  WARN root [2019-03-02 14:44:26,640] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 14:44:26,691] - Components::findLogger: found logger, addr:192.168.0.148:13513
  INFO root [2019-03-02 14:44:26,701] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 14:44:26,701] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 14:44:26,709] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 28936. PID: 15236 ----
  INFO root [2019-03-02 14:44:26,709] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 14:44:26,719] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 14:44:26,720] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 14:44:27,308] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 14:44:27,309] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 14:44:33,503] - Forwarding logs to logger[192.168.0.148:13513/0/0/0]...
 ERROR root [2019-03-02 14:45:30,879] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 14:45:30,892] - The following logs sent to logger failed:
  WARN root [2019-03-02 14:45:30,913] - ==> [2019-03-02 14:45:30 857] TCPPacketReceiver::processPendingEvents(192.168.0.148:13513): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 14:45:30,913] - ==> [2019-03-02 14:45:30 880] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 14:45:30,914] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13566): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 14:45:30,915] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 14:45:30,927] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 14:45:30,928] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 14:45:30,928] - Shutdowner::onShutdownEnd: shutting down
  WARN root [2019-03-02 14:45:30,928] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13637): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  WARN root [2019-03-02 14:45:30,929] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13674): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-03-02 14:45:30,930] - Components::removeComponentByChannel: cellappmgr : 6000, Normal exit!
  WARN root [2019-03-02 14:45:30,931] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13736): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  WARN root [2019-03-02 14:45:30,931] - Baseappmgr::onChannelDeregister: erase baseapp[7001], currsize=0
  INFO root [2019-03-02 14:45:30,932] - Components::removeComponentByChannel: baseapp : 7001, Normal exit!
  WARN root [2019-03-02 14:45:30,932] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13819): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-03-02 14:45:30,932] - Components::removeComponentByChannel: cellapp : 8001, Normal exit!
  WARN root [2019-03-02 14:45:30,933] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13860): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-03-02 14:45:30,933] - Components::removeComponentByChannel: loginapp : 9000, Normal exit!
 DEBUG root [2019-03-02 14:45:30,944] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-03-02 14:45:31,246] - ThreadPool::destroy(): waiting for thread(2)[0x648040,0x6480a8,], try=1
 DEBUG root [2019-03-02 14:45:31,647] - TPThread::~TPThread(): 0x648040
 DEBUG root [2019-03-02 14:45:31,647] - TPThread::~TPThread(): 0x6480a8
 DEBUG root [2019-03-02 14:45:31,647] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-03-02 14:45:31,680] - Bundle::destroyObjPool(): size 12.
 DEBUG root [2019-03-02 14:45:31,680] - Channel::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 14:45:31,681] - TCPPacket::destroyObjPool(): size 11.
 DEBUG root [2019-03-02 14:45:31,681] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2019-03-02 14:45:31,681] - EndPoint::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 14:45:31,681] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 14:45:31,681] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 14:45:31,682] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-03-02 14:45:31,682] - baseappmgr(5000) has shut down.
  INFO root [2019-03-02 14:45:32,684] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [2019-03-02 14:45:57,923] - 
  INFO root [2019-03-02 14:45:57,925] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 14:45:57,930] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 14:45:57,930] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 14:45:57,931] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 14:45:57,943] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:14060 )
  INFO root [2019-03-02 14:45:57,945] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:14060, SOMAXCONN=5.
  INFO root [2019-03-02 14:45:57,945] - server-configs:
  INFO root [2019-03-02 14:45:57,946] - 	internalTcpAddr : 192.168.0.148:14060
  INFO root [2019-03-02 14:45:57,946] - 	componentID : 5000
  WARN root [2019-03-02 14:45:57,955] - invalid UID(0) <= 0, please check UID for environment! automatically set to 34643.
  INFO root [2019-03-02 14:45:58,833] - Components::findLogger: found logger, addr:192.168.0.148:14064
  INFO root [2019-03-02 14:45:58,833] - Components::addComponent: reset orderLog, uid=34643!
  INFO root [2019-03-02 14:45:58,834] - Components::addComponent[logger], uid=34643, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 14:45:58,842] - ---- baseappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: E6918B69E98EAAAB4657C1417BA2685B. Config: Debug X86. Built: 17:20:25 Feb 27 2019. AppID: 5000. UID: 34643. PID: 8540 ----
  INFO root [2019-03-02 14:45:58,844] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 14:45:58,845] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 14:45:58,846] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 14:45:59,800] - ---- baseappmgr is running ----
 DEBUG root [2019-03-02 14:45:59,802] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 14:46:06,189] - Forwarding logs to logger[192.168.0.148:14064/0/0/0]...
 ERROR root [2019-03-02 14:49:22,829] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

